src/poly/matrix.rs

```diff
@@ -101,7 +101,7 @@ pub trait PolyMatrix:
     /// where G^-1(other) is bit decomposition of other matrix
     fn mul_tensor_identity_decompose(&self, other: &Self, identity_size: usize) -> Self;
     /// j is column and return decomposed matrix of target column
-    fn get_column_matrix_decompose(&self, j: usize) -> Self;
+    fn get_column_matrix_decompose(&self, j: usize, base_bits: Option<u32>) -> Self;
```

Comment:
i don't think this function is used with base_bit that is not None. if then can we remove this and set as None